rawkit/raw.py

```diff
@@ -7,6 +7,7 @@
 import random
 import string
 import tempfile
+import numpy as np
```

Comment:
I normally use numpy, so I have very little experience with the regular array. As long as you can write to a normal uint16 array from an iterator numpy isn't needed.